code/index.html

```diff
@@ -2,21 +2,245 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <nav> 
+        <div class="hamburger">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+
+        <img class="logo" src="Penguin_logo.svg" alt="">
+
+        <div class="right-nav">
+          <a href="About">About</a>  
+          <a href="Contact">Contact</a>  
+          <button a href="Get involved">Sign up</a> </button>
```

Comment:
Missing tag alert!
```
<button>
  <a>
  </a>
</button>
```
Or even better - styling the `a` tag to look like a button?